src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1512,7 +1508,7 @@ int wm_vuldet_linux_rm_false_positivies(sqlite3 *db, agent_software *agent, OSHa
         do {
             next = pkg->next;
 
-            if ((pkg->feed & VU_SRC_NVD) && !pkg->discard && pkg->nvd_cond && !pkg->nvd_cond->vulnerable) {
+            if ((pkg->feed == VU_SRC_NVD) && !pkg->discard && pkg->nvd_cond && !pkg->nvd_cond->vulnerable) {
```

Comment:
Is there any possibility to have `pkg->feed` equals to `VU_SRC_NVD | VU_SRC_OVAL`? Just wondering because of the assignment [here](https://github.com/wazuh/wazuh/pull/5035/files#diff-88895483f5107f24a4e15adf36758f39R673).
If that is possible, the `==` comparison will evaluate as true either for `VU_SRC_NVD` as well as `VU_SRC_OVAL`. Thats correct?

Reply:
The idea is to discard only the packages reported by the NVD that have `vulnerable == 0`. If the OVAL and the NVD report the same vulnerability, then we can't discard it even if the NVD says that it is not vulnerable because the OVAL is more precise.

Reply:
Good! This clarifies my question! Thanks!